Library/Homebrew/os/mac/xcode.rb

```diff
@@ -18,12 +18,12 @@ def latest_version
         when "10.10" then "7.2.1"
         when "10.11" then "8.2.1"
         when "10.12" then "9.2"
```

Comment:
Not for poor, poor 10.12?

Reply:
nope. 10.13.2 is minimum macOS to run 9.3